src/Core/src/Handlers/Button/ButtonHandler.Windows.cs

```diff
@@ -17,6 +17,7 @@ protected override void ConnectHandler(Button platformView)
 			_pointerPressedHandler = new PointerEventHandler(OnPointerPressed);
 			_pointerReleasedHandler = new PointerEventHandler(OnPointerReleased);
 
+			platformView.AllowFocusOnInteraction = false;
```

Comment:
I don't think modifying the default platform behavior to fit into the VSM box inside controls is how we should do this. Any decision made against core should be made with the perspective that controls doesn't exist. 

I think we should think of VSM based solutions for this. Maybe focused should be part of a different VSM group? Or perhaps if focused hasn't been defined then it transitions to normal on all three platforms?